Misc/NEWS.d/next/Library/2022-06-14-22-46-05.gh-issue-83638.73xfGK.rst

```diff
@@ -0,0 +1,2 @@
+Add :attr:`sqlite3.Connection.autocommit` for :pep:`249` compliant
+transaction control. Patch by Erlend E. Aasland.
```

Comment:
```suggestion
Add the :attr:`~sqlite3.Connection.autocommit` attribute to
:class:`~sqlite3.Connection` and the *autocommit* parameter to :func:`connect`
to control :pep:`249`-compliant
:ref:`transaction handling <sqlite3-controlling-transactions-autocommit>`.
Patch by Erlend E. Aasland.
```

Similar changes as in the What's New entry. Also, FWIW, "Patch by" sounds a bit archaic and perhaps a little understating the change here; I personally prefer Contributed by for that reason, but that's personal preference.

Reply:
Most NEWS entries seem to use _"Patch by"_, whereas the What's New entries always use _"Contributed by"_. I believe it is an unwritten convention :)

Reply:
Hmm, I've seen both in NEWS entries (whereas only Contributed by in What's New), with "Patch by" originally deriving from the fact that patches were emailed to a mailing list or uploaded to BPO, rather than the modern issue/PR-based workflow. Maybe I'm a younger generation, but to me "Patch" has the specific connotation an atomic change that was sent in to someone else, and specifically that fixes/improves code that already exists rather than contributes something new (which this is), whereas contributing is more generally applicable and characterizes the _role_ of the user involved rather than the concrete nature of the contribution (a patch, versus a commit, PR, or series of such). But again, that's just my personal (over-)interpretation...